src/pages/AcceptInvite.tsx

```diff
@@ -2,45 +2,81 @@ import * as React from "react";
-          src="https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg"
-        />
-      </a>
-      <p style={{ fontSize: 12 }}>
-        Google Play and the Google Play logo are trademarks of Google LLC.
-      </p>
-      <p style={{ fontSize: 12 }}>
-        Apple and the Apple logo are trademarks of Apple Inc., registered in the
-        U.S. and other countries and regions. App Store is a service mark of
-        Apple Inc.
-      </p>
-    </div>
-    <div className="AcceptInviteStep">
-      <h3>
-        <IntlMessage id="accept_invite_step2" />
-      </h3>
-      <a style={{ fontSize: 18 }} href={window.location.href}>
-        {window.location.href}
-      </a>
+const AcceptInvite: React.StatelessComponent<{}> = () => {
+  // Parse out invite token
+  let inviteToken;
+  const query = window.location.search.substring(1);
+  const vars = query.split("&");
+  for (let i = 0; i < vars.length; i++) {
+    const pair = vars[i].split("=");
+    if (decodeURIComponent(pair[0]) === "t") {
+      inviteToken = decodeURIComponent(pair[1]);
+      break;
+    }
+  }
+
+  if (!inviteToken) {
+    return (
+      <div style={{ textAlign: "center" }}>
+        <h3>
+          <IntlMessage id="invalid_accept_invite" />
+        </h3>
+      </div>
+    );
+  }
+
+  // TODO: Pull these schemes from a raha-api/shared?
+  const customSchemeDeeplink = "raha://link/invite?t=" + inviteToken;
+  const httpsSchemeDeeplink = "https://d.raha.app/invite?t=" + inviteToken;
+
+  return (
+    <div className="AcceptInviteSteps">
+      <div className="AcceptInviteStep">
+        <h3>
+          <IntlMessage id="accept_invite_step1" />
+        </h3>
+        <a href="https://play.google.com/store/apps/details?id=app.raha.mobile">
+          <img
+            width="300"
+            alt="Get it on Google Play"
+            src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png"
+          />
+        </a>
+        <br />
+        <a href="https://itunes.apple.com/app/raha/id1434224783?ls=1&mt=8">
+          <img
+            width="250"
+            alt="Download on the Apple App Store"
+            src="https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg"
+          />
+        </a>
+        <p style={{ fontSize: 12 }}>
+          Google Play and the Google Play logo are trademarks of Google LLC.
+        </p>
+        <p style={{ fontSize: 12 }}>
+          Apple and the Apple logo are trademarks of Apple Inc., registered in
+          the U.S. and other countries and regions. App Store is a service mark
+          of Apple Inc.
+        </p>
+      </div>
+      <div className="AcceptInviteStep">
+        <h3>
+          <IntlMessage id="accept_invite_step2" />
+        </h3>
+        <p>
+          <a style={{ fontSize: 18 }} href={httpsSchemeDeeplink}>
+            {httpsSchemeDeeplink}
+          </a>
+        </p>
+        <IntlMessage id="accept_invite_step2_fallback" />
```

Comment:
LGTM, but could we add a comment or something saying why the first link might not work? Are https links less reliable? What if we just showed them the second link?